workflow-service/src/test/java/com/redhat/parodos/workflow/execution/aspect/WorkFlowExecutionAspectTest.java

```diff
@@ -132,10 +134,34 @@ public void ExecuteAroundAdviceWithValidDataTest() {
 		assertEquals(workReport.getStatus().toString(), COMPLETED);
 		assertEquals(workReport.getWorkContext().get(WORKFLOW_DEFINITION_NAME), TEST_WORK_FLOW);
 		assertEquals(workReport.getWorkContext().get(PROJECT_ID), projectID);
+		assertThat(workReport.getError()).isInstanceOf(ResourceNotFoundException.class);
```

Comment:
@eloycoto pls remove this change or replace with
```
assertThat(workReport.getError()).isNull();
```
This is a positive code path that shouldn't end with a failure